README.md

```diff
@@ -60,13 +85,21 @@ will run on the CUDA backend, whereas:
 ```
     SYCL_DEVICE_FILTER=opencl:cpu ./nbodygl
 ```
-will run on a CPU through the OpenCL backend. 
+will run on a CPU through the OpenCL backend. Note the correspondence between options for `SYCL_DEVICE_FILTER` and the output of `sycl-ls`.
 
-**Note**: this is possible because `CMakeLists.txt` specifies building the SYCL code for both CUDA & OpenCL backends:
+**Note**: Selection between DPC++ backends at runtime is possible because `CMakeLists.txt` specifies building the SYCL code for both CUDA (`nvptx64-nvidia-cuda`) & OpenCL (`spir64`) targets:
 ```
      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsycl -fsycl-targets=spir64,nvptx64-nvidia-cuda -fsycl-unnamed-lambda")
 ```
 
+### Selecting the ComputeCpp Host Backend
+
+The experimental ComputeCpp distribution will *only* work with the host backend, and this must be specified with the `COMPUTECPP_TARGET` environment variable:
```

Comment:
Not 100% sure what to say here but if the user downloads ComputeCpp, it should work with either the host device or any USM-supporting OpenCL device (e.g. Intel GPU/CPU).